config/routes.rb

```diff
@@ -16,14 +16,22 @@
   get '/groups/new' => 'groups#new'
   post '/groups/new' => 'groups#create'
 
+  resources :chatrooms, param: :slug
+  resources :messages
+  resources :walks
```

Comment:
Looks like we still have walks as a standalone and we should move that. I want you to see this before I just do it though.